pkgs/servers/web-apps/wiki-js/default.nix

```diff
@@ -1,12 +1,14 @@
-{ stdenv, fetchurl, lib, nixosTests }:
+{ stdenv, fetchFromGitHub, lib, nixosTests }:
 
 stdenv.mkDerivation rec {
   pname = "wiki-js";
   version = "2.5.201";
 
-  src = fetchurl {
-    url = "https://github.com/Requarks/wiki/releases/download/${version}/${pname}.tar.gz";
-    sha256 = "sha256-k2G+jUne/lq0dRJsIQpWlRFg1Nq92bM28YkawKOKlsI=";
+  src = fetchFromGitHub {
+    owner = "Requarks";
+    repo = "wiki";
+    rev = version;
+    sha256 = "pM2sbs6ruzXxg1EiWTzPgj8IORA2Kawe79POlrQFSkc=";
```

Comment:
I'm pretty sure that this one is wrong since it's a tarball which also contains node_modules.